### ISSUE TITLE
A new useless rule created by Kotak Securities. Seems like they want to push clients away.

### ISSUE THREAD

**Issue by algowhizz** [osparamatrix/ks-orderapi-python#53]
We are not allowed to place opposite orders with qty larger than squareoff qty which means that we cannot flip our position in a single order. Way to go Kotak !!
![image](https://user-images.githubusercontent.com/71078804/133195260-3cc37f5c-1e0e-494a-92fc-b44750d58a65.png)

**Comment by WhiteBlackGray** [comment]
> We are not allowed to place opposite orders with qty larger than squareoff qty which means that we cannot flip our position in a single order. Way to go Kotak !!
> ![image](https://user-images.githubusercontent.com/71078804/133195260-3cc37f5c-1e0e-494a-92fc-b44750d58a65.png)

Are you trying to place square off position for MIS position? I think this is how it suppose to work.

**Comment by algowhizz** [comment]
> > We are not allowed to place opposite orders with qty larger than squareoff qty which means that we cannot flip our position in a single order. Way to go Kotak !!
> > ![image](https://user-images.githubusercontent.com/71078804/133195260-3cc37f5c-1e0e-494a-92fc-b44750d58a65.png)
> 
> Are you trying to place square off position for MIS position? I think this is how it suppose to work.

This is not squareoff, I am trying to flip my position from short to long. Currently I have 1 lot short, and want to place an order for 2 lots long. So that I am 1 lot net long.

**Comment by WhiteBlackGray** [comment]
> > > We are not allowed to place opposite orders with qty larger than squareoff qty which means that we cannot flip our position in a single order. Way to go Kotak !!
> > > ![image](https://user-images.githubusercontent.com/71078804/133195260-3cc37f5c-1e0e-494a-92fc-b44750d58a65.png)
> > 
> > 
> > Are you trying to place square off position for MIS position? I think this is how it suppose to work.
> 
> This is not squareoff, I am trying to flip my position from short to long. Currently I have 1 lot short, and want to place an order for 2 lots long. So that I am 1 lot net long.

Yes it means you are squaring off your first leg and creating new positon together right? Is it MIS order type?

**Comment by algowhizz** [comment]
> > > > We are not allowed to place opposite orders with qty larger than squareoff qty which means that we cannot flip our position in a single order. Way to go Kotak !!
> > > > ![image](https://user-images.githubusercontent.com/71078804/133195260-3cc37f5c-1e0e-494a-92fc-b44750d58a65.png)
> > > 
> > > 
> > > Are you trying to place square off position for MIS position? I think this is how it suppose to work.
> > 
> > 
> > This is not squareoff, I am trying to flip my position from short to long. Currently I have 1 lot short, and want to place an order for 2 lots long. So that I am 1 lot net long.
> 
> Yes it means you are squaring off your first leg and creating new positon together right? Is it MIS order type?

Yes, it is MIS and yes thats how flip would work. I have enough margin to place 2 lots long. Ideally, that should be sufficient for Kotak's internal engine to cover existing short and convert it to 1 net long.
